### PR TITLE
[READY] Paginate the blog

### PR DESCRIPTION
**Pull request by RonaldDijkstra**
- [x] set paginate to true
- [x] numbers
- [x] styling 
- [x] styling on page 2 and further
- [x] translate
- [x] mobile
- [x] test

#241

### CODE REVIEW

**Comment by RonaldDijkstra** [comment]
@fatboypunk any idea how to rewrite the class on the body of /blog/2/ and further? It is now: 

`<body class="nl blog blog_2 blog_2_index">`

The last one I need to be just `blog_index`.

**Comment by RonaldDijkstra** [comment]
pagination

**Comment by fatboypunk** [comment]
Not sure, but i can look into it tomorrow.

A solution for now would be to use the the following css:
```css
div[class^="blog_"], div[class*="_index"] {
    color:red 
}
```

As described here http://stackoverflow.com/questions/5110249/wildcard-in-css-for-classes

File: source/blog/index.html.erb [hunk 1]
```diff
@@ -47,10 +39,28 @@ description:
...
-      <% end %>
+    <% if paginate && num_pages > 1 %>
+      <div class="text-center">
+        <ul class="pagination">
+          <% if prev_page %>
+            <li><%= link_to t("blog.prev"), prev_page, class: 'page-step prev' %></li>
+          <% end %>
+
+          <% for p in 1..num_pages %>
+            <% if p == page_number %>
```
Comment (fatboypunk) [inline source/blog/index.html.erb]: `p` - >  `page`

File: source/blog/index.html.erb [hunk 2]
```diff
@@ -47,10 +39,28 @@ description:
...
+              <li class="current"><%= p %></li>
+            <% elsif p == 1 %>
+              <li><%= link_to p, "/blog/" %></li>
+            <% else %>
+              <li><%= link_to p, "/blog/#{p}/" %></li>
+            <% end %>
```
Comment (fatboypunk) [inline source/blog/index.html.erb]: what you could do here is use
```erb
            <% else %>
              append = p == 1 ? nil : "#{p}/"
              <li><%= link_to p, "/blog/#{append}" %></li>
            <% end %>

Comment (RonaldDijkstra) [inline source/blog/index.html.erb]: Like this?

`<% for page in 1..num_pages %>
            <% if page == page_number %>
              <li class="current"><%= page %></li>
            <% elsif p == 1 %>
              <li><%= link_to page, "/blog/" %></li>
            <% else %>
              <% append = page == 1 ? nil : "#{page}/" %>
              <li><%= link_to page, "/blog/#{append}" %></li>
            <% end %>
          <% end %>`

**Comment by RonaldDijkstra** [comment]
Like this?

```erb
          <% for page in 1..num_pages %>
            <% if page == page_number %>
              <li class="current"><%= page %></li>
            <% elsif page == 1 %>
              <li><%= link_to page, "/blog/" %></li>
            <% else %>
              <% append = page == 1 ? nil : "#{page}/" %>
              <li><%= link_to page, "/blog/#{append}" %></li>
            <% end %>
          <% end %>

**Comment by fatboypunk** [comment]
No like 
```erb
          <% for page in 1..num_pages %>
            <% if page == page_number %>
              <li class="current"><%= page %></li>
            <% else %>
              <% append = page == 1 ? nil : "#{page}/" %>
              <li><%= link_to page, "/blog/#{append}" %></li>
            <% end %>
          <% end %>
```

**Comment by RonaldDijkstra** [comment]
Thanks!

**Comment by RonaldDijkstra** [comment]
Seems ready now, thanks for helping @fatboypunk.

**Pull request merged**